docs/sources/reference/api/docker_remote_api_v1.16.md

```diff
@@ -1326,6 +1327,11 @@ Create a new image from a container's changes
              "Cmd":[
                      "date"
              ],
+	     "Labels": [
+                     "Vendor=Acme",
+                     "License=GPL",
+                     "Version=1.0"
+             ],
```

Comment:
@vieux @phemmer Okay, I've just noticed that host labels are ["key=value"] in the API and not a map.  @vieux Why was it chosen to be done this way?  It does make the code a bit wonky.